src/content/docs/fr/tutorial/1-setup/index.mdx

```diff
@@ -0,0 +1,71 @@
+---
+type: tutorial
+unitTitle: Créer et déployer votre premier site Astro
+title: 'Vérification : Unité 1 - Configuration'
+description: >-
+  Tutoriel : Construisez votre premier blog Astro —
+
+  Préparez votre environnement de développement, et créez et déployez votre premier site Astro.
+i18nReady: true
+---
+import Badge from '~/components/Badge.astro';
+import Checklist from '~/components/Checklist.astro';
+import Box from '~/components/tutorial/Box.astro';
+
+
+
+
+Maintenant que vous savez ce que vous allez créer, configurons tous les outils dont vous aurez besoin !
+
+Cette unité vous montre comment configurer votre environnement de développement et déployer sur Netlify. Passez directement à l'[Unité 2](/fr/tutorial/2-pages/) si vous êtes déjà à l'aise avec votre environnement et votre flux de travail.
+
+:::tip[utilisation de StackBlitz]
+
+Vous voulez compléter ce tutoriel dans un éditeur de code en ligne ?
+<details>
+<summary>Suivez ces instructions, puis passez directement à l'Unité 2 !</summary>
+
+**Configurer StackBlitz**
+1. Rendez-vous sur [astro.new](https://astro.new) et cliquez sur le bouton pour ouvrir le modèle "Empty Project" dans StackBlitz.
+
+2. Cliquez sur "Sign in" en haut à droite pour vous connecter avec vos informations GitHub.
```

Comment:
I'll said `identifiants` instead of `informations` to translate `credentials`.  
```suggestion
2. Cliquez sur "Sign in" en haut à droite pour vous connecter avec vos identifiants GitHub.
```

Reply:
LGTM